src/main/java/org/venice/beachfront/bfapi/geoserver/PKIAuthHeaders.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2016, RadiantBlue Technologies, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package org.venice.beachfront.bfapi.geoserver;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile({ "pki-geoserver-auth" })
+public class PKIAuthHeaders extends HttpHeaders implements AuthHeaders {
+
+	private static final long serialVersionUID = 1L;
+
+	@Override
+	public HttpHeaders get() {
+		return this;
```

Comment:
Is this all that needs to happen or is this handled elsewhere?

Reply:
Yes, this is all. This class is mostly a placeholder for when we're not using BasicAuthHeaders.

Reply:
During our next ATO for this 2.0 stuff, we'll eventually have to move to PKI auth. It looks like this code is done already, in this branch, right?